i18n/locales/source/dashboard/data.yml

```diff
@@ -1,663 +1,648 @@
-# See en.yml for general localization information.
```

Comment:
changes to this file are an expected side effect of https://github.com/code-dot-org/code-dot-org/pull/20728